spikeinterface/curation/auto_merge.py

```diff
@@ -311,24 +315,59 @@ def compute_templates_diff(sorting, templates, num_channels=5, num_shift=5, pair
 
     return templates_diff
 
+class MockWaveformExtractor:
```

Comment:
Can't we move this to the core?
It seems like something useful to have even for certain users.

Reply:
At the moment no. I want to find a really better solution.
I did this a a very temporary work around.